Scroller/EditableWrapper.js

```diff
@@ -248,20 +249,23 @@ const EditableWrapper = (props) => {
 				}
 			}
 		}
-	}, [dataSize, addRearrangedItems, removeRearrangedItems]);
+	}, [dataSize, addRearrangedItems, removeRearrangedItems, scrollContainerHandle]);
 
 	const moveItemsByKeyDown = useCallback((ev) => {
 		const {keyCode} = ev;
 		const container = scrollContentRef.current;
 		const {itemWidth, prevToIndex} = mutableRef.current;
-		const toIndex = is('left', keyCode) ? prevToIndex - 1 : prevToIndex + 1;
+		const {rtl} = scrollContainerHandle.current;
 
-		const itemLeft = toIndex * itemWidth - container.scrollLeft;
+		const toIndex = (rtl ^ is('left', keyCode)) ? prevToIndex - 1 : prevToIndex + 1;
```

Comment:
`typeof rtl` could be 'boolean' or 'undefined', so it looks that `rtl != is('left', keyCode)` is also possible, right? If bitwise XOR operator is preferred, a kind of type casting hack is recommended for safety like `!rtl ^ !is('left', keycode)`.
There is a related Q&A on [stackoverflow](https://stackoverflow.com/questions/4540422/why-is-there-no-logical-xor-in-javascript).